GoedeDoelenHelpen/Startup.cs

```diff
@@ -49,7 +49,6 @@ public void ConfigureServices(IServiceCollection services)
                 options.Password.RequireNonAlphanumeric = false;
                 options.Password.RequireUppercase = true;
                 options.Password.RequireLowercase = false;
-                options.Password.RequiredUniqueChars = 6;
```

Comment:
complete away?

Reply:
We already have at least 2 unique chars with a required upper case and required number